tests/Resources/Page/OCPDashboard/InstalledOperators/InstalledOperators.robot

```diff
@@ -62,4 +63,10 @@ Wait Until Uninstallation Completes
 
 Operator Should Be Uninstalled
   [Arguments]  ${operator}
-  Page Should Not Contain Element  //a[@data-test-operator-row="${operator}"]
\ No newline at end of file
+  Page Should Not Contain Element  //a[@data-test-operator-row="${operator}"]
+
+Get RHODS version
+    @{list} =  OpenShiftCLI.Get  kind=ClusterServiceVersion  label_selector=olm.copiedFrom=redhat-ods-operator
```

Comment:
does it need a space between `label_selector= and From=`

Reply:
It does not! :D